configs/envoy_double_proxy_v2.template.yaml

```diff
@@ -60,6 +60,13 @@
           typed_config: {}
         tracing:
           operation_name: INGRESS
+          # Notice that tracing provider configuration is now part of "envoy.filters.network.http_connection_manager" config.
```

Comment:
nit: I think this comment can be skipped, same elsewhere.

Reply:
fixed